packages/terra-props-table/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Added styling to force long Props Table entries to line wrap
```

Comment:
lets be specific to indicate we are referring to wrapping the propsTypes: 
`Added styling to wrap long prop-type 'type' entries`